src/main/twirl/gitbucket/core/settings/hooks.scala.html

```diff
@@ -14,7 +14,7 @@
           <p>
             Webhooks notify external services when certain events occur within this repository.
             When any of the specified events occur, GitBucket will send a POST request to all of the endpoints (URLs) you provide.
-            Learn more about this feature on the <a href="https://github.com/gitbucket/gitbucket/wiki/API-WebHook" target="_blank">GitBucket Wiki</a>.
+            Learn more about this feature on the <a href="https://github.com/gitbucket/gitbucket/wiki/API-WebHook" target="_blank" rel="noopener">GitBucket Wiki</a>.
```

Comment:
`Links to cross-origin destinations are unsafe`
https://web.dev/external-anchors-use-rel-noopener/?utm_source=lighthouse&utm_medium=devtools